transaction-api/src/main/java/com/example/auction/transaction/api/TransactionService.java

```diff
@@ -36,6 +36,8 @@
 
     ServiceCall<PaymentInfo, Done> submitPaymentDetails(UUID itemId);
 
+    ServiceCall<Boolean, Done> submitPaymentStatus(UUID itemId);
```

Comment:
How about using an enum as a pyaload? It would allow extending in the future while keeping backwards compatibility. Boolean is restricting us to `yes/no` while an enum can be extended.